gallery/src/components/NxTreeViewRadioSelect/NxTreeViewRadioSelectPage.tsx

```diff
@@ -5,216 +5,24 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { NxTable, NxP, NxCode } from '@sonatype/react-shared-components';
+import { NxCode, NxTextLink, NxP, NxWarningAlert } from '@sonatype/react-shared-components';
 
-import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
-
-import NxTreeViewRadioSelectExample from './NxTreeViewRadioSelectExample';
-import NxTreeViewRadioSelectCustomTooltipExample from './NxTreeViewRadioSelectCustomTooltipExample';
-import NxTreeViewRadioSelectDisabledExample from './NxTreeViewRadioSelectDisabledExample';
-
-const nxTreeViewRadioSelectExampleCode = require('./NxTreeViewRadioSelectExample?raw'),
-    nxTreeViewRadioSelectDisabledExampleCode = require('./NxTreeViewRadioSelectDisabledExample?raw'),
-    nxTreeViewRadioSelectCustomTooltipExampleCode =
-        require('./NxTreeViewRadioSelectCustomTooltipExample?raw');
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
 
 const NxTreeViewRadioSelectPage = () =>
   <>
     <GalleryDescriptionTile>
+      <NxWarningAlert>
+        <NxCode>NxTreeViewRadioSelect</NxCode> has been renamed to <NxCode>NxCollapsibleItemsRadioSelect</NxCode>
+        and is now deprecated and will be removed in the future.
+        {/* eslint-disable-next-line max-len */}
```

Comment:
Same as in the other file; can be formatted to not need this.

Reply:
> Same as in the other file; can be formatted to not need this.

Updated:
dfc4d3d4a764e73148f71ff5c7851d8916c2f6d6